flatliners/alertcorr.py

```diff
@@ -0,0 +1,122 @@
+from .baseflatliner import BaseFlatliner
+import pandas as pd
+from datetime import datetime
+
+
+class AlertCorrelation(BaseFlatliner):
+    '''
```

Comment:
is this calculating the correlation for a cluster or a version.
you should name the class to reflect that